src/keystore/sdcard.py

```diff
@@ -146,9 +241,18 @@ async def storage_menu(self):
             if menuitem == 255:
                 return
             elif menuitem == 0:
-                if await self.save_mnemonic():
+                filename = await self.get_input()
```

Comment:
What do you think about suggesting the first word of the mnemonic as the default name for the seed?

```suggestion
                filename = await self.get_input(self.mnemonic.split()[0])
```